stores/history.ts

```diff
@@ -0,0 +1,56 @@
+import { atom } from 'nanostores'
+
+import { debounce } from '../lib/time.js'
+import { current, type LchValue } from './current.js'
+
+let $undos = atom<LchValue[]>([])
+let $redos = atom<LchValue[]>([])
+let $prev = atom<LchValue>({ a: 100, c: 0, h: 0, l: 0 })
+let fromUndoRedo = false
+
+current.subscribe(
+  debounce(100, (curr, oldValue) => {
+    if (!oldValue) {
```

Comment:
I think you do not need anymore by moving to:

```js
let $prev = atom<LchValue>(current.get())
```

Reply:
done